app/views/layouts/_attention.html.erb

```diff
@@ -1,22 +1,12 @@
-<% alert_messages = [] %>
-<% handler_errors.each do |error| %>
-  <% alert_messages.push error.translate %>
-<% end %>
-
-<% alert_messages.push(flash[:alert]) if flash[:alert] %>
-
-<% notice_messages = [] %>
-<% notice_messages.push(flash[:notice]) if flash[:notice] %>
-
-<% alert_messages = alert_messages.collect{|msg| msg.gsub("contact support", mail_to("info@openstax.org", "contact support")).html_safe} %>
+<% errors = collect_errors %>
 
```

Comment:
I extracted this into a method in application_helper.  That way the view can call `collect_errors.any?` to see if it should set an error class on inputs.